pkg/handler/v1/project/project.go

```diff
@@ -48,27 +48,60 @@ func CreateEndpoint(projectProvider provider.ProjectProvider) endpoint.Endpoint
 		}
 
 		user := ctx.Value(middleware.UserCRContextKey).(*kubermaticapiv1.User)
+		settings, err := settingsProvider.GetGlobalSettings()
+		if err != nil {
+			return nil, common.KubernetesErrorToHTTPError(err)
+		}
+
+		if !user.Spec.IsAdmin && settings.Spec.UserProjectsLimit != 0 {
```

Comment:
I would check if it is not <=0 because right now negative numbers are not covered by any validation in the API.